test/Transactions/Orleans.Transactions.Tests/Runners/TransactionTestRunnerBase.cs

```diff
@@ -52,10 +52,6 @@ private string GetTestGrainClassName(TransactionTestConstants.TransactionGrainSt
 
             if (grainStates == TransactionTestConstants.TransactionGrainStates.SingleStateTransaction)
                 return TransactionTestConstants.SingleStateTransactionalGrain;
-            if (grainStates == TransactionTestConstants.TransactionGrainStates.DoubleStateTransaction)
```

Comment:
isn't you want to remove the inline on the tests as well? otherwise those tests would still run.

Reply:
oh nvm. I get it